src/views/Error/index.scss

```diff
@@ -0,0 +1,26 @@
+@use "/src/stylesheets/responsiveness";
+
+.error-view {
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    background: linear-gradient(#8296D9,#BFACF0);
+    @include responsiveness.font(10vw, 72px, 144px);
+    .error-view-message {
+        color: var(--color-text-primary);
+        font-family: "Lato";
```

Comment:
We should probably have a fallback font. Another suggestion is to put this in [src/stylesheets/_typography.scss](https://github.com/CruzHacks/cruzhacks-2022-website/blob/development/src/stylesheets/_typography.scss), though this is not a necessity.

Reply:
This doesn't seem to be addressed yet, but we can do that later.

Reply:
That's my error, I forgot to make that update. I will do that now!